main.py

```diff
@@ -0,0 +1,81 @@
+from typing import Type, Optional
+
+from fastapi import FastAPI, Depends, HTTPException
+from sqlalchemy.orm import Session
+
+import crud
+import schemas
+from database import SessionLocal
+
```

Comment:
DB tables are not created automatically. Please, create tables automatically while starting the project.
You can use `models.Base.metadata.create_all(bind=engine)`